internal/build/info.go

```diff
@@ -33,6 +33,8 @@ var info Info
 
 var once sync.Once
 
+var DefaultSemver = "0.0.0-dev"
```

Comment:
This must be a const. As a var, anyone can modify it.

Reply:
PR: https://github.com/databricks/cli/pull/891